### ISSUE TITLE
About Chinese character recognition

### ISSUE THREAD

**Issue by RobinHan24** [TJCVRS/CRNN_Tensorflow#28]
I tried to train a model of Chiese character, at first, I made a dataset which only contains 5 characters, and the size of dataset is 200000. The loss is about 0 and accuracy is about 1, but when I test my real data, the recognition effect is terrible. Anyone can help me? Thank you very much.

**Comment by TJCVRS** [comment]
@RobinHan24 That means the model is terribly overfit you can try to enlarge your dataset or reduce the training epoch

**Comment by githubsongsong** [comment]
why the reduce of training epoch can reduce the overfit, I think the increase can reduce the overfit. Am I true?
